uli_init/simulate.py

```diff
@@ -142,6 +152,13 @@ def quench(
             )
 
             if shrink_kT and shrink_steps:
```

Comment:
do you want a warning or some sort of handling if only one of these values is provided? I am assuming if `shrink_steps` is provided, the user wants to shrink even if they didn't provide `shrink_kT`.

Reply:
Good idea